lib/SimpleSAML/Utils/EMail.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+namespace SimpleSAML\Utils;
+
+use PHPMailer\PHPMailer\PHPMailer;
+use PHPMailer\PHPMailer\Exception;
+
+use SimpleSAML\Configuration;
+use SimpleSAML\Logger;
+use SimpleSAML\XHTML\Template;
+
+/**
+ * E-mailer class that can generate a formatted e-mail from array
+ * input data.
+ *
+ * @author Jørn Åne de Jong, Uninett AS <jorn.dejong@uninett.no>
+ * @package SimpleSAMLphp
+ */
+
+class EMail
+{
+
+    /**
+     * Get the default e-mail address from the configuration
+     * This is used both as source and destination address
+     * unless something else is provided at the constructor.
+     *
+     * It will refuse to return the SimpleSAMLphp default address,
+     * which is na@example.org.
+     *
+     * @return string Default mail address
+     */
+    public static function getDefaultMailAddress() {
+        $config = Configuration::getInstance();
+        $address = $config->getString('technicalcontact_email', 'na@example.org');
+        if ('na@example.org' === $address) {
+            throw new \Exception('technicalcontact_email not set');
```

Comment:
To prevent confusion: maybe make more explicit inthe message that it is in fact set but not changed from default value